testing/integration/upgrade_test.go

```diff
@@ -208,18 +211,18 @@ func TestStandaloneUpgrade(t *testing.T) {
 
 			parsedUpgradeVersion, err := version.ParseVersion(define.Version())
 			require.NoErrorf(t, err, "define.Version() %q cannot be parsed as agent version", define.Version())
-			testStandaloneUpgrade(ctx, t, agentFixture, parsedVersion, parsedUpgradeVersion, "", true)
+			testStandaloneUpgrade(ctx, t, agentFixture, parsedVersion, parsedUpgradeVersion, "", true, true, false, "")
```

Comment:
The values of skip-verify depends on the versions being tested. Since this test will run also for older sets of versions we should keep the check

```suggestion
			skipVerify:=version_8_7_0.Less(parsedVersion)
			testStandaloneUpgrade(ctx, t, agentFixture, parsedVersion, parsedUpgradeVersion, "", skipVerify, true, false, "")
```